src/containers/Articles/Articles.js

```diff
@@ -50,12 +23,13 @@ const Articles = () => {
         {posts.map(post => (
           <ArticlesItem
             title={post.title}
-            body={`${post.body.slice(0, 150)}...`}
-            date={post.date}
-            link={`/article/${post.link}`}
+            body={`${post.body.slice(0, 155)}...`}
```

Comment:
I guess this 155 is the maximum of characters to see in a body column, can you please put this in a constant with a proper name?